### ISSUE TITLE
Doesn't work properly on Google Images

### ISSUE THREAD

**Issue by Saectar** [extesy/hoverzoom#892]
Although in general Hover Zoom+ works correctly in Google Images, when displaying the information box it turns out that the image can no longer be enlarged; also Hover Zoom+ also Hover Zoom+ does not work on related images from Google Images.

![13_03_2022 (11-22-34 AM) - %t](https://user-images.githubusercontent.com/4337724/158069805-558bae6f-8dee-477d-94d5-1063596f834f.png)

Url: https://www.google.com/
Windows 10 Pro
Hover Zoom+: 1.0.189
Browser: Mozilla Firefox 98.0

**Comment by Saectar** [comment]
It still doesn't work properly. The zoom in the suggested images of Google Images sometimes load and sometimes not.

**Comment by GrosPoulet** [comment]
Hi @Saectar 
On Chrome, HZ+ works fine for Google Images.
On FF, HZ+ stops zooming images when you scroll through results for a long time & works inconsistently on related images.
This issue is related to Google Images plug-in, since i do not observe this issue on Bing.
I have no idea where it comes from, @extesy any hint ?

**Comment by extesy** [comment]
Sorry, I have no clue. The differences between browsers create the worst kind of bugs which are hardest to debug. My bet would be on some Firefox's quirk because its engine is less tested due to their team size and relative popularity of Firefox vs Chrome.

**Comment by GrosPoulet** [comment]
It looks like HZ+ analysis is not triggered when user scrolls or clicks an image for details :disappointed:

**Comment by extesy** [comment]
Maybe related to #890 and the bug is caused by using deprecated `windowOnDOMNodeInserted`?

**Comment by GrosPoulet** [comment]
Yes, this `windowOnDOMNodeInserted` is becoming the Usual Suspect 😅

**Comment by Saectar** [comment]
Oh, too bad they still don't know what causes HoverZoom to work inconsistently in Google Images. Hopefully they will be able to fix it at some point.
